ztp/internal/data/cluster/files/add-internal-ip-bmh.sh

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/bash
 
 # This script will only be present in nodes that don't have an internal NIC. For those nodes we need
-# to add the internal IP to the external NIC.
-external_dev="{{ .ExternalNIC.Name }}"
+# to add the internal IP to the external `br-ex` bridge used by OVS:
 internal_ip="{{ .InternalIP }}"
-nmcli connection modify "${external_dev}" +ipv4.addresses "${internal_ip}" ipv4.method auto
-ip addr add "${internal_ip}" dev "${external_dev}"
\ No newline at end of file
+nmcli connection modify br-ex +ipv4.addresses "${internal_ip}" ipv4.method auto
+ip addr add "${internal_ip}" dev br-ex
```

Comment:
We need the device here because during the boot and installation phase `br-ex` doesn't exist and it's not up.

TBH, I would recommend sticking with the current script and not changing the logic. We will optimize later on